### PR TITLE
Update update.md to be relevant to Prism 8.0

### PR DESCRIPTION
**Pull request by Haskellered**
> @TheKodeToad 
> if you pr to https://github.com/PrismLauncher/bot/pulls i promise to review it :) (maybe)

### CODE REVIEW

**Review by Trial97 — APPROVED** [review]

**Comment by TheKodeToad** [comment]
i was too slow

File: tags/update.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ color: blue
 aliases: ['updating', 'autoupdate']
 ---
 
-Windows auto-updating is WIP. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
+Windows auto-updating is only supported on 8.0+. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
```
Comment (TheKodeToad) [inline tags/update.md:7]:
```suggestion
Windows auto-updating is only supported on 8.0+. For now, you will need to download the installer and run it again in order to update. You will not lose your instances.
```

For now? It's the past now

Maybe you could say "if you are still on a previous version"?

**Review by TheKodeToad — CHANGES_REQUESTED** [review]

File: tags/update.md (hunk 1 above)
Comment (Haskellered) [reply tags/update.md:7]: Oh yeah I'll make some changes.

Comment (Haskellered) [reply tags/update.md]: How about now?

**Review by TheKodeToad — APPROVED** [review]
I totally didn't commit so I am definitely allowed to approve this

**Comment by TheKodeToad** [comment]
Thank you!

**Pull request merged**